libraries/AP_Motors/AP_MotorsHeli_Swash.cpp

```diff
@@ -42,7 +42,6 @@ const AP_Param::GroupInfo AP_MotorsHeli_Swash::var_info[] = {
     // @Description: This linearizes the swashplate servo's mechanical output to account for nonlinear output due to arm rotation.  This requires a specific setup procedure to work properly.  The servo arm must be centered on the mechanical throw at the servo trim position and the servo trim position kept as close to 1500 as possible. Leveling the swashplate can only be done through the pitch links.  See the ardupilot wiki for more details on setup.
     // @Values: 0:Disabled,1:Enabled
     // @User: Standard
-    AP_GROUPINFO("LIN_SVO", 3, AP_MotorsHeli_Swash, _linear_swash_servo, 0),
```

Comment:
If this is deleted then the parameter meta data should be deleted too?

Reply:
@MattKear Hope you don't mind that i removed the rest of the metadata.  Be sure to pull this change back into you local branch.